circle.yml

```diff
@@ -0,0 +1,29 @@
+machine:
+  php:
+    # https://circleci.com/docs/environment#php
+    version: 5.6.22
+  environment:
+    # DB config. Using default CircleCI's database.
+    TERMINUS_ENV: ci-$CIRCLE_BUILD_NUM
+    TERMINUS_SITE: wp-redis
+
+dependencies:
+  cache_directories:
+    - ~/.composer/cache
+  pre:
+    # Set the PHP timezone so that Behat script does not fail.
```

Comment:
> This change is one I made last week after the fallout of switching to ubuntu 14.04. It feels like a hacky way to kill two birds with one stone. Any better ideas?

Nothing great. Given it's a problem specific to Circle though, putting it in the YAML file makes most sense.